pkg/activator/handler/concurrency_reporter_test.go

```diff
@@ -425,3 +427,61 @@ func revisionInformer(ctx context.Context, revs ...*v1.Revision) {
 		revisions.Informer().GetIndexer().Add(rev)
 	}
 }
+
+func BenchmarkConcurrencyReporter(b *testing.B) {
+	ctx, cancel, _ := rtesting.SetupFakeContextWithCancel(b)
+	defer cancel()
+
+	// Buffer equal to the activator.
+	reqCh := make(chan network.ReqEvent, 100)
+	cr := NewConcurrencyReporter(ctx, activatorPodName, reqCh, make(chan []asmetrics.StatMessage, 1000))
```

Comment:
how do we know we never write more than 1000 messages to statCh? I think this works because we write 1 first time we see the revision (= 100), and reportCh only runs every second after that. So this is only safe as long as the benchmark finishes within a second, I _think_. Are we ok with that?

Reply:
True, let's be super safe.

Reply:
Pushed.